site/en/blog/third-party-cookie-deprecation-trial/index.md

```diff
@@ -0,0 +1,221 @@
+* Steps to reproduce a broken user-facing experience must be provided. In particular, this should be an experience for the user operating the device where the cookie is stored, and not a user performing later analysis of data. If we cannot validate a broken user experience then the **registration request will be rejected**.
+* Otherwise the **registration request will be approved**.
+
+We plan to offer an appeals process if the registering origin believes more information could clarify a review decision. The registrant can request an appeal by reapplying on the OT console. The intent of appeals is for requests that were rejected due to missing the requested information (known breakage bug and/or breakage repro steps) and/or if the registering origin believes more information could satisfy these requirements to clarify a review decision. 
+
+
+## Apply for the deprecation trial
+
+{% Aside %}
+
+Before applying, you must report the functionality that will be broken by the third-party cookie deprecation, at [goo.gle/report-3pc-broken](https://goo.gle/report-3pc-broken).
+
+{% endAside %}
+
+Include reproduction steps that our team can use to verify the functional breakage. Alternatively if it’s easier and/or your functionality is gated by login or similar, you can provide a link to a recording of the steps to reproduce the problem, using [Chrome DevTools Recorder](/docs/devtools/recorder/).
+
+
+
+1. Starting the week of November 27, 2023 navigate to the "3PCD Deprecation Trial" via the [list of Active Trials on the Chrome Origin Trials page](/origintrials/#/trials/active) and click "Register".
+2. For "Web Origin" you should provide the origin which serves your embedded page or scripts.
+3. The "Third-party matching" option will depend on how you need to provide the token. The options are explained in more detail in [Add the trial token](#add-the-trial-token) below.
+    * If you are providing the token via HTTP header or meta tag on your own embedded pages, **do not check** "Third-party matching".
+    * If you are injecting the token via JavaScript into a different site, you **must check** "Third-party matching".
+    * If you need to do both, you will need to make separate registrations.
+4. If you host cross-site content across multiple subdomains, then tick the "match all subdomains" option.
+    * Tokens will match multiple subdomains similarly to wildcard matching, e.g. `*.<domain>`. Request a token for `example.com` and it can be provided on `a.example.com`, `b.example.com`. Third-party cookie access will still only be re-enabled for the specific origins that provide the token, not all the subdomains. See [What cookies are enabled when subdomain matching is enabled?](#what-cookies-are-enabled-when-subdomain-matching-is-enabled) below.
```

Comment:
```suggestion
    * Tokens will match multiple subdomains similarly to wildcard matching, e.g. `*.<domain>`. Request a token for `example.com` and it can be provided on `a.example.com`, `b.example.com`. Third-party cookie access will still only be re-enabled for the specific origins that provide the token, not all the subdomains. See [What cookies are enabled when subdomain matching is enabled?](#what-cookies-are-enabled-when-subdomain-matching-is-enabled).
```